lib/davinci_dtr_test_kit/profiles/questionnaire_response/questionnaire_response_create.rb

```diff
@@ -8,10 +8,12 @@ class QuestionnaireResponseCreateTest < Inferno::Test
     description 'A sever SHALL support the QuestionnaireResponse create interaction'
 
     id :questionnaire_response_create
+    optional
     input :create_questionnaire_resources,
```

Comment:
create and update inputs still need titles so underscores don't show through

Reply:
Addressed in [first read/search->read/search step, input titles, wording](https://github.com/inferno-framework/davinci-dtr-test-kit/pull/35/commits/898f1471140e63899323346b0d3296d48f7af929)